bpf/tests/Makefile

```diff
@@ -53,7 +53,7 @@ clean:
 	rm -f $(wildcard *.i)
 	rm -f $(wildcard *.d)
 
-BPF_TEST_FLAGS:=
+BPF_TEST_FLAGS:= $(GO_TEST_FLAGS)
```

Comment:
@ti-mo not sure if there was a particular reason this Makefile wasn't using the global test flags we declare in the top of the agent, but all I'm trying to do here is get `-json` conditionally based on the same logic as already exists in the top-level `Makefile.defs` declaration. Do you have any reservations about this?

I also toyed with the idea of using `$(GO_TEST)` below instead, but since this target typically runs in a container, I found that causes other problems.

Reply:
@joestringer No particular reason, go ahead!